torch/fx/experimental/symbolic_shapes.py

```diff
@@ -439,26 +439,26 @@ def evaluate_guards_for_args(self, *args):
         # and wrap_fake_symbolic
         meta_converter = MetaConverter()
         pytree.tree_map_only(torch.Tensor, partial(meta_converter, shape_env=new_env), args)
-        return all(guard.xreplace(new_env.var_to_val) == value for guard, value, _ in self.guards)
+        return all(guard.xreplace(new_env.var_to_val) for guard, _ in self.guards)
```

Comment:
how come you can drop the value test

Reply:
I removed values altogether from the guards.